### ISSUE TITLE
Screenreader cannot see popup element descriptor

### ISSUE THREAD

**Issue by SteveMarvin** [performant-software/faircopy#357]
Found an outlier...when in the element tray, the reader notes the i button as "button" and although clicking space will open the info popup, the screenreader doesn't see it to read it
![image](https://user-images.githubusercontent.com/18423533/146036959-ae07328a-5941-4ec5-b434-f3676512afc5.png)
